runner/app/pipelines/image_to_image.py

```diff
@@ -235,12 +221,35 @@ def __call__(
                 # Default to 8step
                 kwargs["num_inference_steps"] = 8
 
+        # trying different cases for prompt_embed because some models dont support without pooled_prompt_embeds.
         try:
-            outputs = self.ldm(prompt, image=image, **kwargs)
-        except torch.cuda.OutOfMemoryError as e:
-            raise e
+            compel_proc = Compel(tokenizer=self.ldm.tokenizer, text_encoder=self.ldm.text_encoder)
```

Comment:
@JJassonn69 I think something went wrong during a rebase. I can not find Compel in the code 🤔.

Reply:
compel==2.0.3 should be included in the requirements.txt, might have been removed mistakenly.

Reply:
fixed, included the compel in the requirements and installed the dependencies to check for any conflicts, no issues.